app/scripts/reports/detail/reportsdetail.html

```diff
@@ -92,8 +92,8 @@ <h3 class="panel-title"><span class="navbar-brand" translate>Resultate</span>
             <tr>
                 <td ng-repeat="col in $columns" ng-bind-html="renderedTitles(col.field)"></td>
             </tr>
-            <tr ng-repeat="dataEntry in $data | filter: search.query.$">
-              <td ng-repeat="col in $columns" ng-bind-html="col.getValue(this, dataEntry)"></td>
+            <tr ng-repeat="dataEntry in result.entries | filter: search.query.$">
+              <td ng-repeat="col in $columns" ng-bind-html="getValue(col.field,dataEntry, result.entries)"></td>
```

Comment:
getValue does not have a third param, right?

Reply:
done